packages/patternlab/styleguide/source/_patterns/02-molecules/callout-link~download.json

```
@@ -0,0 +1,12 @@
+{
+  "calloutLink": {
+    "linkTemplate": "download",

```

```suggestion
    "linkTemplate": "download",
```
can we make this a boolean value? It seems like this field should have two values, is download or is not download.